easybuild/easyblocks/c/cp2k.py

```diff
@@ -816,3 +830,12 @@ def sanity_check_step(self):
         }
 
         super(EB_CP2K, self).sanity_check_step(custom_paths=custom_paths)
+
+    def make_module_extra(self):
+        """Set up a CP2K_DATA_DIR environment variable to find CP2K provided basis sets"""
+
+        txt = super(EB_CP2K, self).make_module_extra()
+        if os.path.exists(os.path.join(self.installdir, 'data')):
+            txt += self.module_generator.set_environment('CP2K_DATA_DIR',
+                                                          os.path.join(self.installdir, 'data'))
```

Comment:
@hajgato if you need a value multiple times, it's cleaner to just construct it once:

```python
data_dir = os.path.join(self.installdir, 'data')
if os.path.exists(data_dir):
    txt += self.module_generator.set_environment('CP2K_DATA_DIR', data_dir)
````